tuna/utils/db_utility.py

```diff
@@ -153,6 +158,85 @@ def session_retry(session, callback, actuator, logger=LOGGER):
   return False
 
 
+def gen_update_query(obj, attribs, tablename):
+  """Create a query string updating all attributes for the input object"""
```

Comment:
This is a generic function. It will create an update query on the table with 'tablename' for the supplied 'obj', in which it will set the fields declared in 'attribs'.